### PR TITLE
Add Stacks and KeyValueTags integration tests

### PR DESCRIPTION
**Pull request by dlpzx**
### Feature or Bugfix
Implement tests for Stacks and KeyValueTags api calls (inside core/stacks) as part of https://github.com/data-dot-all/dataall/issues/1220

### Relates
- #1220 

### Security
Please answer the questions below briefly where applicable, or write `N/A`. Based on
[OWASP 10](https://owasp.org/Top10/en/).

- Does this PR introduce or modify any input fields or queries - this includes
fetching data from storage outside the application (e.g. a database, an S3 bucket)?
  - Is the input sanitized?
  - What precautions are you taking before deserializing the data you consume?
  - Is injection prevented by parametrizing queries?
  - Have you ensured no `eval` or similar functions are used?
- Does this PR introduce any functionality or component that requires authorization?
  - How have you ensured it respects the existing AuthN/AuthZ mechanisms?
  - Are you logging failed auth attempts?
- Are you using or adding any cryptographic features?
  - Do you use a standard proven implementations?
  - Are the used keys controlled by the customer? Where are they stored?
- Are you introducing any new policies/roles/users?
  - Have you used the least-privilege principle? How?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by dlpzx** [comment]
Tested locally:
image

File: tests_new/integration_tests/core/stack/test_stack.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        },
+    ).contains(
+        'UnauthorizedOperation',
+        'SAVE_KEY_VALUE_TAGS',
+        'Duplicate tag keys found',
+    )
+
+
+def test_update_list_key_value_tags_delete_tags(client1, session_env1, session_id):
+    # Test delete tag
```
Comment (noah-paige) [inline tests_new/integration_tests/core/stack/test_stack.py:90]: should we check that key value tags are created on this environment to ensure they are being updated to be deleted? 

Either using a fixture to ensure tag is created to the environment or call the lines of code in `test_update_list_key_value_tags_add_tag` and `test_update_list_key_value_tags_delete_tags` in the same test to ensure this test is not dependent on others

**Review by noah-paige — COMMENTED** [review]
One minor comment - then think this is good

File: tests_new/integration_tests/core/stack/test_stack.py (hunk 1 above)
Comment (dlpzx) [reply tests_new/integration_tests/core/stack/test_stack.py:90]: Yeep, let's do it!

**Review by noah-paige — APPROVED** [review]
tested and looks good!

**Pull request merged**